3/debian-10/rootfs/opt/bitnami/scripts/zookeeper/entrypoint.sh

```diff
@@ -5,7 +5,9 @@
 set -o errexit
 set -o nounset
 set -o pipefail
-# set -o xtrace # Uncomment this line for debugging purposes
+if ${BITNAMI_DEBUG:-false}; then
+	set -o xtrace
+fi
```

Comment:
I think it's safer and more robust to compare the value as string. It also fixes the indentation.

```suggestion
if [[ "${BITNAMI_DEBUG:-false}" != "false" ]]; then
    set -o xtrace
fi
```